src/main/java/cart/auth/BasicAuthorizationExtractor.java

```diff
@@ -0,0 +1,36 @@
+package cart.auth;
+
+import cart.dto.auth.AuthInfo;
+import org.apache.tomcat.util.codec.binary.Base64;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class BasicAuthorizationExtractor {
+
+    private static final String AUTHORIZATION = "Authorization";
+    private static final String BASIC_TYPE = "Basic";
+    private static final String DELIMITER = ":";
+    private static final int CREDENTIALS_EMAIL_INDEX = 0;
+    public static final int CREDENTIALS_PASSWORD_INDEX = 1;
+
+    public AuthInfo extract(HttpServletRequest request) {
+        String header = request.getHeader(AUTHORIZATION);
```

Comment:
```suggestion
        String header = request.getHeader(HttpHeaders.AUTHORIZATION);
```
이미 정의된 상수들을 사용할수도 있겠네요!
Basic도 아마 있을거에요

Reply:
HTTP와 관련된 상수들은 기본 스프링 프레임워크 패키지에서 제공을 한다는 것을 처음 알았습니다. 감사합니다. 😄 

그런데 알려주신 Basic 상수 값은 찾아보니 스프링 시큐리티 패키지에 선언되어 있는 것 말고는 없다고 나오더라구요... 상수 값을 사용하기 위해 패키지를 import 하는 것은 뭔가 배보다 배꼽이 커지는 것 같아 일단 Basic은 상수로 유지하는 것으로 결정했습니다!